api/openapi/specs/api.yaml

```
@@ -433,6 +468,45 @@ components:
           additionalProperties:
             $ref: '#/components/schemas/ResourceStats'
           description: A map of resource names to their corresponding statistics
+    InspectHostnames:
+      type: object
+      title: InspectHostnames
+      description: A list of hostnames
+      required: [total, items]
+      properties:
+        total:
+          type: integer
+          example: 200
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/InspectHostname'
+    InspectHostname:
+      type: object
+      title: InspectHostname
+      description: A list of zones in which the hostname is available in
```

This description seems to be the one of the `zones` field not of this entire object.
I think this is more
```suggestion
      description: An supported hostname along with the zones it exists in
```